driver-core/src/main/com/mongodb/client/model/vault/EncryptOptions.java

```diff
@@ -173,12 +175,12 @@ public String getQueryType() {
     /**
      * The RangeOptions
      *
-     * <p>It is an error to set RangeOptions when the algorithm is not "RangePreview".
+     * <p>It is an error to set RangeOptions when the algorithm is not "Range".
      * <p>Note: The Range algorithm is experimental only. It is not intended for public use. It is subject to breaking changes.
      * @param rangeOptions the range options
      * @return this
      * @since 4.9
-     * @mongodb.server.release 6.2
+     * @mongodb.server.release 8.0
      * @mongodb.driver.manual /core/queryable-encryption/ queryable encryption
      */
     @Beta(Beta.Reason.SERVER)
```

Comment:
Can the `@Beta` annotation can now be removed?

Reply:
The range algorithm is currently marked as unstable as part of this ticket. There is a ticket [DRIVERS-2776](https://jira.mongodb.org/browse/DRIVERS-2776) intended to promote 'range' to GA. We should consider removing the `@Beta` annotation once the drivers ticket progresses to the 'teams implementing' phase.